docs/reference/search/retriever.asciidoc

```diff
@@ -194,7 +194,7 @@ GET /index/_search
                 }
             ],
             "rank_constant": ...
-            "window_size": ...
+            "rank_window_size": ...
```

Comment:
I don't think we can change an API in a bug fix release. While this is an experimental API, making a breaking change in a bug fix is a no go.

Does this have an effect on the pagination fix other than just making its backport easier?

Reply:
Yeah makes sense. No effect really, just making backport more straightforward with already verified commits. Maybe we could keep the new classes but revert the `rank_window_size` renaming, so that the end-user API remains the same for 8.14.2?